### PR TITLE
Add support for rentals contents overrides endpoint

### PR DESCRIPTION
**Pull request by StoneFrog**
Already documented:
http://developers.bookingsync.com/reference/endpoints/rentals_contents_overrides/

### CODE REVIEW

**Comment by thilonel** [comment]
@StoneFrog is this done? If so, request a review please :)

**Comment by StoneFrog** [comment]
@thilonel 
It is (otherwise would be WIP), but since everyone can review it I didn't feel like pinging someone specific. But since you already volunteered... 😄

File: lib/bookingsync/api/client/rentals_contents_overrides.rb [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+      # @param options [Hash] A customizable set of options.
+      # @option options [Array] fields: List of fields to be fetched.
+      # @return [Array<BookingSync::API::Resource>] Array of rentals_contents_overrides.
+      #
+      # @example Get the list of rentals contents overrides for the current account
+      #   rentals_contents_overrides = @api.rentals_contents_overrides
+      #   rentals_contents_overrides.first.description.en # => "Overriden English description"
+      # @see http://developers.bookingsync.com/reference/endpoints/rentals_contents_overrides/#list-rentals-contents-overrides
+      def rentals_contents_overrides(options = {}, &block)
+        paginate :rentals_contents_overrides, options, &block
```
Comment (thilonel) [inline lib/bookingsync/api/client/rentals_contents_overrides.rb]: Nothing tests that these options are passed and usable and do stuff

File: spec/bookingsync/api/client/rentals_contents_overrides_spec.rb [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+      end
+    end
+  end
+
+  describe ".rentals_contents_override", :vcr do
+    let(:prefetched_rentals_contents_override_id) {
+      find_resource("#{@casette_base_path}_rentals_contents_overrides/returns_rentals_contents_overrides.yml", "rentals_contents_overrides")[:id]
+    }
+
+    it "returns rentals_contents_override" do
```
Comment (thilonel) [inline spec/bookingsync/api/client/rentals_contents_overrides_spec.rb]: This only tests that the right endpoint is being requested, nothing about what's returned.
It's misleading when the `it` states it's testing something but it actually isn't.

File: lib/bookingsync/api/client/rentals_contents_overrides.rb [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+      end
+
+      # Get a single rentals_contents_override
+      #
+      # @param rentals_contents_override [BookingSync::API::Resource|Integer]
+      # rentals_contents_override or ID of the rentals_contents_override.
+      # @return [BookingSync::API::Resource]
+      # @see http://developers.bookingsync.com/reference/endpoints/rentals_contents_overrides/#get-a-single-rentals-content-override
+      def rentals_contents_override(rentals_content_override)
+        get("rentals_contents_overrides/#{rentals_content_override}").pop
```
Comment (thilonel) [inline lib/bookingsync/api/client/rentals_contents_overrides.rb]: No specs for accepting resource as argument, only for ID. No spec for what it's returning.

File: lib/bookingsync/api/client/rentals_contents_overrides.rb (hunk 1 above)
Comment (StoneFrog) [reply lib/bookingsync/api/client/rentals_contents_overrides.rb]: This is kind of tough to test, as according to docs it currently doesn't use any additional args.

File: lib/bookingsync/api/client/rentals_contents_overrides.rb (hunk 3 above)
Comment (StoneFrog) [reply lib/bookingsync/api/client/rentals_contents_overrides.rb]: Accepting resource as argument is already tested inside resource by checking that "#{resource}" == resource.id.to_s

**Pull request merged**